### PR TITLE
Do not apply selector to helper

### LINKED ISSUE #7: Extra selectors added on `css` properties
Example:
```
const foo = css` 
width: 0px;
`;

const bar = styled`
color: green;
${foo}
`
```

Results in two classes:
```
.asdas.asdas.asdas.asdas {
width: 0px;
}
.asdas.asdas {
color: green;
}
```

expected behavior:
```
.asdas.asdas {
width: 0px;
color: green;
}
```

Note, that further nesting will add even more selectors.

### PR DESCRIPTION
**Pull request by dmk255**
Fixes #7.

I've also added a unit test.

Note, I also branched off of PR https://github.com/QuickBase/babel-plugin-styled-components-css-namespace/pull/6

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1 +1,2 @@
 output.js
+lib/*
```
Comment (nvanselow) [inline .eslintignore]: Nice catch!

File: src/visitors/cssNamespace.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default (path, state) => {
   const cssNamespace = getCssNamespace(state);
 
   if (
-    (isStyled(path.node.tag, state) || isHelper(path.node.tag, state)) &&
+    isStyled(path.node.tag, state) &&
```
Comment (nvanselow) [inline src/visitors/cssNamespace.js]: Minor change: Can you also remove `isHelper` from the imports from the top as it is no longer used?

It looks like we are missing the eslint rule for that. I can add that in a follow up PR so we don't have to catch these manually.

Comment (dmk255) [reply src/visitors/cssNamespace.js]: yup I'll do that now

**Comment by nvanselow** [comment]
Thanks again for your PRs @dmk255. Version 0.0.9 should be up on npm now.

**Comment by dmk255** [comment]
Awesome! I appreciate the quick turn around. If I catch anything else as I use it I'll do my best to make PRs for them. 

Thanks for open sourcing this tool!

**Comment by danielhusar** [comment]
Has this been fixed? I can still see duplicated rules when using `css`.

**Comment by danielhusar** [comment]
Adding `path.node.tag.name !== 'css' &&` to this condition fixes it https://github.com/QuickBase/babel-plugin-styled-components-css-namespace/blob/70918fe2962694b505e586ebfa14abe83a2b5c74/src/visitors/cssNamespace.js#L48

**Pull request merged**